src/Features/LanguageServer/ProtocolUnitTests/References/FindAllReferencesHandlerTests.cs

```diff
@@ -150,9 +198,22 @@ private static void AssertValidDefinitionProperties(LSP.ReferenceItem[] referenc
                 }
 
                 Assert.Null(referenceItems[i].DefinitionText);
+                Assert.Equal(0, referenceItems[i].DefinitionIcon.ImageId.Id);
                 Assert.Equal(definitionId, referenceItems[i].DefinitionId);
                 Assert.NotEqual(definitionId, referenceItems[i].Id);
             }
         }
+
+        private sealed class ProgressCollector<T> : IProgress<object>
```

Comment:
Making a note here to switch this as well in https://github.com/dotnet/roslyn/issues/48350